src/app/components/Header/Header.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import React, { ReactElement } from "react";
+import { usePathname } from "next/navigation";
+
+import { NavLabel, NavIconButton } from "./Header.types";
+import MobileHeaderNav from "../MobileHeaderNav/MobileHeaderNav";
+import DesktopHeaderNav from "../DesktopHeaderNav/DesktopHeaderNav";
+
+function Header(): ReactElement {
+  const pathname = usePathname();
+  const [isMobileMenuOpen, setIsMobileMenuOpen] = React.useState(false);
+  const isActiveLink = (linkPathname: string): boolean => linkPathname === pathname;
+
+  const navLabels: NavLabel[] = [
```

Comment:
navLabels and navIconButtons would be better to move to some config.ts or similar
or at least define them outside (above it) of the component